pkg/controller/storagecluster/controller_test.go

```diff
@@ -1443,8 +1443,10 @@ func TestStoragePodGetsScheduled(t *testing.T) {
 	require.Len(t, podControl.Templates, 2)
 	podTemplate1 := expectedPodTemplate.DeepCopy()
 	podTemplate1.Spec.NodeName = "k8s-node-1"
+	podTemplate1.Labels["k8s-node-running-px-pod"] = "k8s-node-1"
```

Comment:
nit: Use the constant here so we don't have to change code everywhere if we end up changing the key.